wire-ios-data-model/Source/ManagedObjectContext/NSManagedObjectContext+zmessaging.m

```diff
@@ -486,11 +499,11 @@ - (void)enqueueDelayedSaveWithGroup:(ZMSDispatchGroup *)group;
     }];
 }
 
-- (NSPersistentStore *)firstPersistentStore
+- (nullable NSPersistentStore *)firstPersistentStore
 {
     NSArray *stores = [self.persistentStoreCoordinator persistentStores];
-    NSAssert(stores.count == 1, @"Invalid number of stores");
-    NSPersistentStore *store = stores[0];
+    NSAssert(stores.count <= 1, @"Invalid number of stores");
```

Comment:
it could happen we get here and we already torn the stack, but we should never have two stores